statistics/feedback.go

```diff
@@ -772,10 +770,7 @@ func EncodeFeedback(q *QueryFeedback) ([]byte, error) {
 	var buf bytes.Buffer
 	enc := gob.NewEncoder(&buf)
 	err = enc.Encode(pb)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-	return buf.Bytes(), nil
+	return buf.Bytes(), err
```

Comment:
Should we wrap the `err` with `errors.Trace(err)`, since it comes from a standard lib without stack information?